src/subclass/object.rs

```diff
@@ -320,7 +306,31 @@ unsafe impl<T: ObjectSubclass> IsSubclassable<T> for ObjectClass {
     }
 }
 
-pub trait ObjectImplExt: ObjectImpl + ObjectSubclass {
+pub trait ObjectImplExt {
+    /// Chain up to the parent class' implementation of `glib::Object::constructed()`.
+    ///
+    /// Do not override this, it has no effect.
```

Comment:
Well, it can't be overridden anymore so this part of the docs can go away :)

Reply:
Done! :)